### PR TITLE
New UI translations

### PR DESCRIPTION
**Pull request by kelson42**
New wave of UI translations

### CODE REVIEW

File: core/src/main/res/values-ar/strings.xml [hunk 1]
```diff
@@ -129,23 +116,58 @@
...
+  <string name="go_to_next_page">انتقل إلى الصفحة التالية</string>
+  <string name="table_of_contents">جدول المحتويات</string>
+  <string name="select_languages">اختر اللغات</string>
+  <string name="save_languages">حفظ اللغات</string>
+  <string name="send_feedback">إرسال تعليق</string>
+  <string name="expand">وسع</string>
+  <string name="history">تاريخ</string>
+  <string name="history_from_current_book">التاريخ من الكتاب الحالي</string>
+  <string name="search_history">سجل البحث</string>
+  <string name="selected_items">تم تحديد %1$d</string>
```
Comment (macgills) [inline core/src/main/res/values-ar/strings.xml]: I would say this is wrong
Correct example 
```
<string name="welcome_messages">Hello, %1$s! You have %2$d new messages.</string>
```

File: core/src/main/res/values-as/strings.xml [hunk 2]
```diff
@@ -37,4 +32,5 @@
     <item>@string/help_18</item>
     <item>@string/help_19</item>
   </string-array>
+  <plurals name="books_count">
```
Comment (macgills) [inline core/src/main/res/values-as/strings.xml]: not a valid `plurals`

File: core/src/main/res/values-be/strings.xml [hunk 3]
```diff
@@ -1,40 +1,28 @@
...
   <string name="menu_bookmarks">Закладкі</string>
-  <string name="menu_random_article">Выпадковы артыкул</string>
-  <string name="menu_full_screen">Поўны экран</string>
-  <string name="menu_exit_full_screen">Выйсці з рэжыму поўнага экрану</string>
   <string name="menu_read_aloud">Чытаць уголас</string>
   <string name="menu_read_aloud_stop">Спыніць чытанне ўголас</string>
   <string name="save_media">Захаваць медыя як…</string>
   <string name="save_media_error">Узнікла памылка пры захаванні медыя-файла!</string>
-  <string name="save_media_saved">Медыя захавана як %1$s у тэчцы вашых выяваў</string>
+  <string name="save_media_saved">Saved media as %s to Android/media/org.kiwix…/</string>
```
Comment (macgills) [inline core/src/main/res/values-be/strings.xml]: this is in English?

File: core/src/main/res/values-ar/strings.xml (hunk 1 above)
Comment (kelson42) [reply core/src/main/res/values-ar/strings.xml]: I'm not so sure about the regex to match the %... stuff? Do you know what it should be?

File: core/src/main/res/values-as/strings.xml (hunk 2 above)
Comment (kelson42) [reply core/src/main/res/values-as/strings.xml]: Indeed, and this is going to be a bit harder to manage.

File: core/src/main/res/values-be/strings.xml (hunk 3 above)
Comment (kelson42) [reply core/src/main/res/values-be/strings.xml]: The point here is hat a `%s` is expected... and we don't get it, so we rollback to English. What is the correct behaviour?

File: core/src/main/res/values-ar/strings.xml (hunk 1 above)
Comment (macgills) [reply core/src/main/res/values-ar/strings.xml]:
```
// copied from java.util.Formatter
// %[argument_index$][flags][width][.precision][t]conversion
private static final String formatSpecifier
    = "%(\\d+\\$)?([-#+ 0,(\\<]*)?(\\d+)?(\\.\\d+)?([tT])?([a-zA-Z%])";
```

File: core/src/main/res/values-be/strings.xml (hunk 3 above)
Comment (macgills) [reply core/src/main/res/values-be/strings.xml]: anything that matches the pattern I posted should be preserved in a translation

**Pull request closed without merging**